bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -606,6 +611,7 @@ data:
         ${multi_network_enabled_flag} \
         ${multi_network_policy_enabled_flag} \
         ${admin_network_policy_enabled_flag} \
+        ${persistent_ips_enabled_flag} \
```

Comment:
I think we need this in cluster manager, not ovnkube-controller. So here
https://github.com/openshift/cluster-network-operator/blob/4506c8216425dd92211a3a95071fae8afebe41a3/bindata/network/ovn-kubernetes/managed/ovnkube-control-plane.yaml#L168
and here
https://github.com/openshift/cluster-network-operator/blob/4506c8216425dd92211a3a95071fae8afebe41a3/bindata/network/ovn-kubernetes/self-hosted/ovnkube-control-plane.yaml#L128

Reply:
I think you are still missing it in 
 https://github.com/openshift/cluster-network-operator/blob/4506c8216425dd92211a3a95071fae8afebe41a3/bindata/network/ovn-kubernetes/self-hosted/ovnkube-control-plane.yaml#L128

Reply:
Oh right !

My bad, I had missed those were two different files. 

Thanks.